_inc/client/components/data/query-site-benefits/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { fetchSiteBenefits, isFetchingSiteBenefits } from 'state/site';
+import { isDevMode } from 'state/connection';
```

Comment:
Could you clarify why we need to bring in `isDevMode` here? The modal will not be displayed anywhere when in development mode, right?

Reply:
leftover from other changes, I will get rid of it